assessment__new-hire--app-dev/src/components/day.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class='calendar__day'>
-        <div class='title'> {{ dateObj.toDateString() }} </div>
+        <div class='title'> {{ dateObj.toUTCString().split(" ").slice(0,4).join(" ") }} </div>
```

Comment:
this is a bit too much JS in the html, this might need its own method that returns a value that is placed here